client/components/dashboard/dashboard-directive.html

```diff
@@ -10,8 +10,7 @@
                ng-model="container.model"
                ng-click="clickContainer($event, container)">
       <div ng-show="container.model.container.header_text"
-           class="pk-container-header">
-        {{ container.model.container.header_text }}
+           class="pk-container-header" btf-markdown="container.model.container.header_text">
```

Comment:
angular-sanitize is used by the angular-markdown component.  Will add a comment to clarify this.